pkg/api/login.go

```diff
@@ -146,7 +109,7 @@ func LoginPost(c *m.ReqContext, cmd dtos.LoginCommand) Response {
 
 	user := authQuery.User
 
-	loginUserWithUser(user, c)
+	hs.loginUserWithUser(user, c)
```

Comment:
I think we can use AuthTokenService directly here.

Reply:
Probably. I want to move the AuthTokenService middleware to the middleware package eventually so maybe wait with this?

Reply:
yes. lets bring this up post merge.